Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBoxes/BoundingBoxGizmoHandleHandMotionTypeEnum.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX
+{
+    public enum BoundingBoxGizmoHandleHandMotionType
+    {
+        handRotatesToRotateObject,
+        handMovesToRotateObject
+    }
+}
```

Comment:
I believe the file name suffix needs to match `type` as well.

Reply:
this file has been deleted.